### ISSUE TITLE
smapi installer is not work 

### ISSUE THREAD

**Issue by jar100** [MartyrPher/SMAPI-Android-Installer#26]
hello 
i want use this app 

I followed this video.
 https://www.youtube.com/watch?v=jOXl6Dmu7wY&feature=youtu.be

but smapi installer is not work

SMAPI Stardew Valley is not installed

i am downloaded stardewvalley to the Galaxy Store. 

In my opinion smapi installer not found my stardewvalley app

**Comment by MartyrPher** [comment]
Yeah, the installer currently doesn't work with the Galaxy Store version. I have looked into it in the past but there is something wacky going on in the `WriteApk.java` file. I might have to try a manual install again to see if it works with that.

**Comment by lannytlop** [comment]
Ihave same problem. 
![Screenshot_20200202-034414_SMAPI Android Installer](https://user-images.githubusercontent.com/60552549/73597345-50a60c80-456e-11ea-8e4e-97b49fd465ec.jpg)

Galaxy store

Now, please ...
Is there a solution?
